src/register/constants/organisationAddress.ts

```diff
@@ -27,69 +27,79 @@ export const organisationAddress = {
+              validationError: {
+                value: 'Enter Building and street',
+                controlId: 'officeAddressOne',
+              },
+              control: 'officeAddressOne',
+              classes: '',
+            },
           },
-          control: 'officeAddressOne',
-          classes: '',
-        },
-      },
-      {
-        input: {
-          validationError: {
-            value: 'Enter the length of hearing in minutes, for example "20"',
-            controlId: 'officeAddressTwo',
-          },
-          control: 'officeAddressTwo',
-          classes: '',
-        },
-      },
-      {
-        input: {
-          label: {
-            text: 'Town or city',
-            classes: 'govuk-label--m',
+          {
+            input: {
+              validationError: {
+                value: 'Enter Building and street line 2 of 2',
+                controlId: 'officeAddressTwo',
+              },
+              control: 'officeAddressTwo',
+              classes: '',
+            },
           },
-          control: 'townOrCity',
-          validators: ['required'],
-          validationError: {
-            value: 'Enter town or city',
-            controlId: 'townOrCity',
+          {
+            input: {
+              label: {
+                text: 'Town or city',
+                classes: 'govuk-label--m',
+              },
+              control: 'townOrCity',
+              validators: ['required'],
+              validationError: {
+                value: 'Enter town or city',
+                controlId: 'townOrCity',
+              },
+              classes: 'govuk-!-width-two-thirds',
+            },
           },
-          classes: 'govuk-!-width-two-thirds',
-        },
-      },
-      {
-        input: {
-          label: {
-            text: 'County',
-            classes: 'govuk-label--m',
+          {
+            input: {
+              label: {
+                text: 'County',
+                classes: 'govuk-label--m',
+              },
+              control: 'county',
+              classes: 'govuk-!-width-two-thirds',
+            },
           },
-          control: 'county',
-          classes: 'govuk-!-width-two-thirds',
-        },
-      },
-      {
-        input: {
-          label: {
-            text: 'Postcode',
-            classes: 'govuk-label--m',
-          },
-          control: 'postcode',
-          validators: ['required'],
-          validationError: {
-            value: 'Enter enter postcode',
-            controlId: 'Poscode',
-          },
-          classes: 'govuk-input--width-10',
-        },
+          {
+            input: {
+              label: {
+                text: 'Postcode',
+                classes: 'govuk-label--m',
+              },
+              control: 'postcode',
+              validators: ['required'],
+              validationError: {
+                value: 'Enter postcode',
+                controlId: 'Postcode',
```

Comment:
Needs to be `'postcode'` with a lowercase 'p' as that's what the `control` is on line 93.

Reply:
sorry; changed it to postcode